### DISCUSSION TITLE
[fukasawa主题] 侧边栏支持开关伸缩

### DISCUSSION THREAD

**Discussion by Muzych** [tangly1024/NotionNext#2073] [Ideas]
**为什么提出这个新的特性改动**
希望能够在使用fukasawa主题，支持侧边栏的伸缩

**描述一下你推荐的解决方案**
可以添加一个和黑暗模式相同的伸缩按钮，控制侧边栏的伸缩

**Comment by tangly1024** [comment]
放在哪个位置合适呢

**Comment by Muzych** [comment]
> 放在哪个位置合适呢

我感觉可以和dark mode并排放，一排放两个按钮。
